packages/ember-self-focused/tests/integration/components/self-focused-test.js

```diff
@@ -40,13 +41,17 @@ module('Integration | Component | self-focused', function(hooks) {
       </div>
     `);
 
-    selfFocusedDiv = this.element.querySelector('#container > div');
+    requestAnimationFrame(() => {
```

Comment:
This would not be needed if the main service sets up a test waiter. The issue with doing this in tests is that **all** tests (even in consuming apps) will have to do this.

Reply:
+1

Reply:
I removed `requestAnimationFrame(() => {})` usage form all the test other than the last one. I could not get it pass w/o it.